tests/e2e/dpa_deployment_suite_test.go

```diff
@@ -703,7 +711,14 @@ var _ = Describe("Configuration testing for DPA Custom Resource", func() {
 			if dpa.Spec.Configuration.Restic != nil && dpa.Spec.Configuration.Restic.PodConfig != nil {
```

Comment:
In this file, there are still old style restic configuration for DPA (`grep -Iinr '\.restic' tests`). I would update them to use nodeAgent syntax

Reply:
Yes. I left the above intentionally. We are removing the old configuration in 1.4, so this change is not about removing it. We should still have test to cover what should be functional and it's going to be removed later.

Reply:
Agree
I would just update some logs that still say restic instead of Node Agent then  (`grep -Iinr 'log.Printf("Waiting for restic' tests`)